src/View/Helper/Form.php

```diff
@@ -64,8 +64,9 @@ public function render(FormInterface $form): string
 
         $formContent = '';
 
-        $renderer = $this->getView();
+        $renderer = $this->view;
```

Comment:
Can we guarantee that `$this->view` is not null here? Unless the view is provided to the constructor, I'd say not.

Reply:
This change is a BC break.

Reply:
reverted

Reply:
@mimmi20 to explain the BC part. This is a BC break only in a strict sense: this method have a possibility of being overridden in extending classes. I do not see a reason to do so. And impact would be absolutely minuscule if someone is affected. However ` getView()` is a part of helper interface and this components was out for a long time to ensure any kind of unexpected usages could be encountered. Unless it is a part of bug fix or the change has some significant advantage it would be better to avoid it.